showcase/src/assets/examples/table/with-footer/table.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, ViewEncapsulation } from "@angular/core";
+import { StarkPaginationConfig, StarkTableColumnProperties, StarkTableFilter } from "@nationalbankbelgium/stark-ui";
+
+const DUMMY_DATA: object[] = [
+	{ id: 1, cost: 12, description: "number one" },
+	/* ... */
+	{ id: 44, cost: 6, description: "the twelfth description" }
+];
+
+@Component({
+	selector: "showcase-table-with-footer",
+	templateUrl: "./table-with-footer.component.html",
+	styleUrls: ["./table-with-footer.component.scss"],
+	/* tslint:disable-next-line:use-view-encapsulation */
```

Comment:
Remove this line as it is not relevant for the example code

Reply:
done

Reply:
> Great! It is working now! Just small remarks and also... don't forget to write unit tests for this new feature 😉

Yes sure, I'll create an issue for this purpose : https://github.com/NationalBankBelgium/stark/issues/1613